### DISCUSSION TITLE
Remove shorts, put back trending

### DISCUSSION THREAD

**Discussion by TxJK** [TeamVanced/VancedManager#596] [Ideas]
Don't know if it's a YouTube feature introduced in certain region but this shorts option should be removed and I need the good old trending button back.
Please.![IMG_20210626_124216.jpg](https://user-images.githubusercontent.com/86510486/123505356-33e70900-d67c-11eb-97b4-507ee0a2ff42.jpg)

**Comment by X1nto** [comment]
not related to manager

**Reply by l3ugl2oono0** [reply]
Thx
